src/current/v24.3/configure-replication-zones.md

```diff
@@ -59,19 +65,257 @@ In addition, CockroachDB stores internal [**system data**]({% link {{ page.versi
+
+However, there are nuances in how inheritance works that are not explained by this example. For more details about how zone config inheritance works, see the section [How zone config inheritance works](#how-zone-config-inheritance-works).
+
+### How zone config inheritance works
+
+As [discussed previously](#level-priorities), CockroachDB always uses the most granular replication zone available for each schema object (database, table, etc.). More-specific settings applied to schema objects at lower levels of the inheritance tree will always override the settings of objects above them in the tree. All configurations will therefore be modified versions of the [`default` range]({% link {{ page.version.version }}/configure-replication-zones.md %}#view-the-default-replication-zone), which acts as the root of the tree. This means that in practice, **any changes to a specific schema object's zone configurations are by definition user-initiated**, either via [Multi-region SQL abstractions]({% link {{ page.version.version }}/multiregion-overview.md %}) or [manual changes](#why-manual-zone-config-management-is-not-recommended).
+
+Because each zone config inherits all of its initial values from its parent object, **it only stores the values that differ from its parent**. Any fields that are unset will be looked up in the parent object’s zone configuration. This continues recursively up the inheritance hierarchy all the way to the `default` zone config. In practice, most values are cached for performance.
+
+For more information, see the following subsections:
+
+- [The zone config inheritance hierarchy](#the-zone-config-inheritance-hierarchy)
+- [Zone config inheritance - example SQL session](#zone-config-inheritance-example-sql-session)
+- [Why manual zone config management is not recommended](#why-manual-zone-config-management-is-not-recommended)
+
+#### The zone config inheritance hierarchy
+
+The hierarchy of inheritance for zone configs can be visualized using the following outline-style diagram, in which each level of indentation denotes an inheritance relationship.
+
+The only exception to this simple inheritance relationship is that due to a known limitation, sub-partitions do not inherit their values from their parent partitions. Instead, sub-partitions inherit their values from the parent table. For more information, see [cockroachdb/cockroach#75862](https://github.com/cockroachdb/cockroach/issues/75862).
+
+```
+- default
+  - database
+    - table
+      - index
+        - partition (row) 1
+          - sub-partition 1.1 (NB. Sub-partitions inherit from tables, *not* partitions - see cockroachdb/cockroach#75862)
+            - sub-partition 1.1.1
+          - sub-partition 1.2
+    - table
```

Comment:
nit: i believe adding 2 tables to this diagram is trying to illustrate how inheritance comes in when multiple of the same type of object have their zone configured (which is great), but would we be able to make this distinction more clear?

right now, i think i'm reading "table" as "all tables" if that makes why i was confused a bit more clear

Reply:
ah sorry, ya i can see how that is confusing

took a shot at adding some names to the DBs/table/indexes , PTAL - i don't know if these are good names but they attempt to show the hierarchy a little. def open to feedback tho

```
- default
  - database A
    - table A.B
      - index A.B.1
        - partition (row) 1
          - sub-partition 1.1 (NB. Sub-partitions inherit from tables, *not* partitions - see cockroachdb/cockroach#75862)
            - sub-partition 1.1.1
          - sub-partition 1.2
    - table A.C
      - index A.C.1
  - database D
    - ...
```